mediaEvents.js

```diff
@@ -179,6 +179,11 @@ var MediaEvents = (function () {
                 type: 'segment',
                 premature: true
             });
+
+            // Reset state at end of video
+            this.lastKnownPlaybackTime = 0;
```

Comment:
Yep noticed that but thought it safer to leave that as is in case anything else going on.  At least I know my stuff resets correctly at the end of the 'end' event